src/styles/_catalog.scss

```diff
@@ -1,22 +1,26 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
+  grid-template-columns: repeat(4, product-card(width));
+  justify-content: center;
+  gap: product-card(spacing);
   max-width: 944px;
   margin: 0 auto;
```

Comment:
don't give margins and max-width to BEM-block

Reply:
I didn't write these lines of code, they were given to me from the get-go :,(
` max-width: 944px;
  margin: 0 auto;`